modules/core/client/directives/pagination.client.directive.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+//Pagination Directive
+angular.module('core').directive('meanPagination', function () {
+    return {
+        restrict: 'AEC',
+        scope: {
+            meanPaginationData: '=meanPaginationData',
+            meanPaginationItemCount: '=meanPaginationItemCount',
+            meanPaginationUrl: '@meanPaginationUrl',
+            meanPaginationSearch: '@meanPaginationSearch'
+        },
+        controller: ['$scope', '$filter', function($scope, $filter) {
+
+            $scope.buildPager = function () {
+                $scope.pagedItems = [];
+                $scope.itemsPerPage = $scope.meanPaginationItemCount;
+                $scope.currentPage = 1;
```

Comment:
For some reason I think that angular boostrap ui needs it to be current page 1 to start.  Otherwise I am guessing the first page may be called 0